SPECS/xcp-ng-release-linstor.spec

```diff
@@ -19,8 +21,12 @@ sed -i "s/\# \(global_filter\)[[:space:]]*=.*/\1 = [ \"r|^\/dev\/drbd.*|\" ]/g"
 
 %files
 %config(noreplace) %{_sysconfdir}/yum.repos.d/xcp-ng-linstor.repo
+%config(noreplace) %{_sysconfdir}/modprobe.d/drbd.conf
```

Comment:
`%config(noreplace)` means if users modify the configuration future changes from the RPM regarding this file will not be applied (a .rpmnew file will be created instead). Is this what you want, or do you prefer to overwrite the file if you need to bring modifications to it in the future? If you chose to do so, a warning on top of the file will be appropriate.

Reply:
I don't want to overwrite the file, especially if a user needs particular settings. It doesn't seem impossible to me.

Reply:
Couldn't we offer cascading configuration files instead, Like, our defaults in `00-drbd.conf` and user settings or overrides in another file?

Reply:
We can't, it's a kernel module config, we can just use this file.

Reply:
Solved by putting the file in /usr/lib/modprobe.d/ which is the place for default distro configuration, and leaves /etc to the user.